src/js/components/Markdown/__tests__/__snapshots__/Markdown-test.tsx.snap

```diff
@@ -113,8 +113,8 @@ exports[`Markdown renders 1`] = `
   border-bottom: solid 1px rgba(0, 0, 0, 0.12);
   padding-left: 12px;
   padding-right: 12px;
-  padding-top: 6px;
-  padding-bottom: 6px;
+  padding-top: 12px;
```

Comment:
Changes only due to jsdom, does not appear in browser. Correct value is re-assigned below